composer.json

```diff
@@ -14,11 +14,11 @@
     "license": "MIT",
     "require": {
         "php": "^7.3 || ^8.0",
-        "brianium/paratest": "^6.3",
+        "brianium/paratest": "^6.4.2",
         "pestphp/pest-plugin": "^1.0"
     },
     "require-dev": {
-        "pestphp/pest": "dev-master",
+        "pestphp/pest": "^1.21",
```

Comment:
@nunomaduro, should this be changed? 🤔 Not entirely sure.

Reply:
I needed to change that because composer complained at install when using `pest/pest:dev-master`
```
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - brianium/paratest v6.4.3 requires phpunit/phpunit ^9.5.14 -> satisfiable by phpunit/phpunit[9.5.14, ..., 9.5.x-dev].
    - brianium/paratest v6.4.2 requires phpunit/phpunit ^9.5.13 -> satisfiable by phpunit/phpunit[9.5.13, ..., 9.5.x-dev].
    - You can only install one version of a package, so only one of these can be installed: phpunit/phpunit[dev-master, 9.5.13, ..., 9.5.x-dev].
    - phpunit/phpunit 10.0.x-dev is an alias of phpunit/phpunit dev-master and thus requires it to be installed too.
    - pestphp/pest dev-master requires phpunit/phpunit 10.0.x-dev -> satisfiable by phpunit/phpunit[10.0.x-dev (alias of dev-master)].
    - Root composer.json requires pestphp/pest dev-master -> satisfiable by pestphp/pest[dev-master].
    - Root composer.json requires brianium/paratest ^6.4.2 -> satisfiable by brianium/paratest[v6.4.2, v6.4.3].
```

Reply:
Ah, that makes sense. I guess everything is geared up for Pest 2.x 🤔

Reply:
I did think that, does it have a branch alias for 1.x?

Reply:
hmm, seems not

![image](https://user-images.githubusercontent.com/8792274/158259004-a560c5db-969e-47c5-bea5-4de578d6c34f.png)

but fixes can be tagged on that branch

Reply:
@fabio-ivona, I've updated the branch alias on the `1.x` branch. @bastien-phi, would you be able to update this to `dev-1.x`? 🤔

Reply:
can't use dev-1.x but used 1.x-dev

```
  Problem 1
    - Root composer.json requires pestphp/pest dev-1.x, found pestphp/pest[dev-higher-order-test-expect-escape, dev-filtered_datasets, dev-master, dev-next, dev-repeat, dev-revert-462-revert-461-json_fix, dev-better-tap-type-hinting, dev-performs_no_expectations, dev-parallel-highligher-support, dev-parallel, v0.1.0, ..., v0.3.19, v1.0.0, ..., 1.x-dev, 2.x-dev (alias of dev-next)] but it does not match the constraint.
```

Reply:
Yeah, my bad. Got the alias the wrong way round. 👍🏻

Reply:
Yeah indeed... 🤦
I will change that. Good catch !

Reply:
@fabio-ivona Done !

Reply:
We'll need to merge this back into `master` after as well then. 👍🏻 As both 2.x and 1.x will need this fix.